crates/bevy_color/Cargo.toml

```
@@ -16,6 +16,7 @@ bevy_reflect = { path = "../bevy_reflect", version = "0.14.0-dev", features = [
 serde = "1.0"
 thiserror = "1.0"
 wgpu = { version = "0.19.1", default-features = false }
+encase = { version = "0.7", features = ["glam"] }
```

```suggestion
encase = { version = "0.7", default-features = false }
```

`glam` isn't required for the `ShaderType` implementations, so we might as well thin this out.

---

Yep, this will work fine for our use case: when compiled together the set of needed feature flags will be OR'd together.

---

Ah, yeah, good catch, I just pasted the one from bevy_render 😅